storage/tree/suffix_test.go

```diff
@@ -255,7 +265,6 @@ func Test8BitSuffixCache(t *testing.T) {
 		// 8 bits suffix should be cached.
 		{b: 8, path: []byte{0x76}, wantCache: true},
 		// above 8 bits should not be cached.
-		{b: 9, path: []byte{0x40}, wantCache: false},
```

Comment:
This was an incorrect `Suffix` as `path` has to have 2 bytes instead of 1.

Reply:
Well spotted.